middleware/auth.js

```diff
@@ -191,6 +191,25 @@ function verifyAppKey(apiKey, cb) {
 }
 
 
+/**
+ * Return a valid region string for the input string
+ *
+ * @param {Function} cb
+ * @api private
+ */
+function getValidRegionString(region) {
+  switch (region) {
+    case 'EU':
```

Comment:
Should we do case insensitive checks?

Reply:
I integrated case-insensitive checks now.